planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_utils.hpp

```diff
@@ -24,8 +25,14 @@
 
 namespace behavior_path_planner
 {
+using behavior_path_planner::PlannerData;
```

Comment:
The line is added because the unit test can't compile due to `-fpermissive` when it is not included.